cli/testdata/server-config.yaml.golden

```diff
@@ -33,6 +33,10 @@ networking:
     # directly in the database.
     # (default: <unset>, type: bool)
     disablePasswordAuth: false
+    # If enabled, users can switch from password based authentication to oauth based
+    # authentication by logging into an oidc account with the same email address.
+    # (default: <unset>, type: bool)
+    enableOauthAuthConversion: false
```

Comment:
@bpmct do we need to add a new option for this? Is there any problem with this being default instead? It feels bad to add a new config option just for this.

Reply:
Happy to drop this if that is the conclusion.

Reply:
Actually I think we should keep this atm.

This flag is hidden because we need to follow up this PR. See "Decisions to be made" section in the description.

I want to get this PR in as it's been hung up for awhile. Those decisions can come after this. Then we can drop this flag.

Reply:
Maybe I should make it an experiment. Ideally the follow up is quick.

Reply:
I made this an experiment. Just in review terms, adding these adds more re-reviewing. If it's an experiment, I can do quick follow ups. As long as it's an experiment, it will not just idle and get lost imo.